style/preview.less

```diff
@@ -23,17 +24,23 @@
 		}
 
 		&-wordmark {
-			background-image: inline-svg( '../images/wikipedia-wordmark.svg' );
+			background-image: inline-svg( '../images/wikipedia-wordmark-en.svg' );
 			background-repeat: no-repeat;
-			background-color: #eaf3ff;
 			background-position: left;
 			background-position-x: 11px;
 			height: 100%;
 			flex-grow: 1;
 			font-size: 24px;
-			padding-left: 13px;
+			margin-left: 13px;
```

Comment:
background-position-x already have offset 11px, and here gives extra offset, just check on the zeplin, it should have offset left 13px

so either change background position x to 13, and no need for margin left, or the opposite.

Reply:
I did that because the browser I was testing with doesn't support `background-position-x` (FF48) but looking at caniuse, I think this is supported enough that we can use it.